client/src/components/popular.jsx

```diff
@@ -1,18 +1,42 @@
 import React from 'react'
+import $ from "jquery";
 import PhotoList from './photoList.jsx';
 import PopHeader from './header.jsx';
-import style from '../style.css'
+import style from '../style.css';
 
 
 class Popular extends React.Component {
     constructor(props) {
         super(props)
-        this.state = {}
+        this.state = {
+            list: []
+        }
 
         this.scrollLeft = this.scrollLeft.bind(this)
         this.scrollRight = this.scrollRight.bind(this)
+        // this.testcall = this.testcall.bind(this)
     }
 
+componentDidMount() {
+    $.ajax({
+    method: 'GET',
+    url: '/munch-popular',
+    success: (data) => {
+        // console.log(data)
+        this.setState(() => {
+            return {list: data};
+          });
+    }
+ });
+}
+
+// (state) => {
+//     return {quantity: state.quantity + 1};
+//   });
+
+// var state = Object.assign({}, this.state);
+// this.state = data;
+// this.setState(state);
```

Comment:
What does the above commented out code do? Do you need it anymore? Also, don't forget to clear up console logs eventually

Reply:
Oh I wanted to delete that as well :).  It was one way I tried to set up the state.

Reply:
Thank You!